core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/EnvironmentV4Controller.java

```diff
@@ -55,7 +55,7 @@ public SimpleEnvironmentResponse delete(Long workspaceId, String environmentCrn)
 
     @Override
     public DetailedEnvironmentResponse edit(Long workspaceId, String environmentCrn, @NotNull EnvironmentEditRequest request) {
-        return environmentClientService.edit(environmentCrn, request);
+        throw new UnsupportedOperationException("Editong an environment is not supported anymore!");
```

Comment:
why we changed this?

Reply:
It's in cloudbreak and noone calls this. It's left here to not to break the int test code, so it will be easier to rewire them